### ISSUE TITLE
Question: running self-host version

### ISSUE THREAD

**Issue by bingzhangdai** [Eugeny/tabby-web#84]
I would like to run tabby-web on a private network. To simplify the deployment, could I achieve the following?

1. deploy tabby-web with embedded SQLite without dedicated DB service
2. run tabby without auth, since I already have a reverse proxy which handles the authentication

Thanks.

**Comment by gitget0** [comment]
RE your first question - here's an example I found:

```
  tabby:
    image: ghcr.io/eugeny/tabby-web:master
    container_name: tabby
    restart: always
    ports:
      - 9190:80
    volumes:
      - /home/XXX/docker/Tabby/data:/app-dist
    environment:
      - PORT=80
      - DEBUG=True
      - ENABLE_HOMEPAGE=False
      - DATABASE_URL=sqlite:////app-dist/database.sqlite3
      - APP_DIST_STORAGE=file:///app-dist
      - SOCIAL_AUTH_GITHUB_KEY=XXX
      - SOCIAL_AUTH_GITHUB_SECRET=XXX
```

As for your second question:

`2. run tabby without auth, since I already have a reverse proxy which handles the authentication`

I also use tabby internally, and to depend on external auth to be able to use tabby internally seems.. overkill.

Would also love an option with local auth if anyone figured out a way 👍

**Comment by t-lessy** [comment]
Yeah, both suggestions would be nice, in my opinion! Maybe, regular terminal user (in my mind) would be like to see sqlite option with a little migration guide? I don't like to mess with dumps, when I just need to move my Compose stack. And sqlite is suits for me, as it moves like a file, but also suitable for small home-server w/o need to read/write whole tens of megabytes into db.

All good,
Antony!
